lib/_debugger.js

```diff
@@ -1588,7 +1588,7 @@ Interface.prototype.repl = function() {
   this.repl.on('exit', exitDebugRepl);
 
   // Set new
-  this.repl.eval = this.debugEval.bind(this);
+  this.repl.eval = () => this.debugEval();
```

Comment:
Can you make sure you find the rest of the functions in this file and make sure you're passing all the args in these spots?